invenio_accounts/admin.py

```
@@ -0,0 +1,147 @@
+
+from flask import current_app, flash
+from flask_admin.actions import action
+from flask_admin.contrib.sqla import ModelView
+from flask_admin.form.fields import DateTimeField
+from flask_babelex import gettext as _
+from flask_login import login_user, logout_user
+from werkzeug.local import LocalProxy
+
+from .cli import commit
+from .models import Role, User
+
+_datastore = LocalProxy(lambda: current_app.extensions['security'].datastore)
+
+
+class UserView(ModelView):
+    """Flask-Admin view to manage users."""
+
+    can_view_details = True
+    can_delete = False
+
+    list_all = (
+        'id', 'email', 'active', 'confirmed_at', 'last_login_at',
+        'current_login_at', 'last_login_ip', 'current_login_ip', 'login_count'
+    )
+
+    column_list = \
+        form_columns = \
+        column_searchable_list = \
+        column_sortable_list = \
+        column_details_list = \
+        list_all
+
+    column_filters = ('id', 'email', 'active', 'confirmed_at', 'last_login_at',
+                      'current_login_at', 'login_count')
+
+    column_default_sort = ('last_login_at', True)
+
+    form_overrides = {
+        'last_login_at': DateTimeField
+    }
+
+    column_labels = {
+        'current_login_ip': _('Current Login IP'),
+        'last_login_ip': _('Last Login IP')
+    }
+
+    @action('inactivate', _('Inactivate'),
+            _('Are you sure you want to inactivate selected users?'))
+    @commit
+    def action_inactivate(self, ids):
+        """Inactivate users."""
+        try:
+            count = 0
+            for user_id in ids:
+                user = _datastore.get_user(user_id)
+                if user is None:
+                    raise ValueError(_("Cannot find user."))
+                if _datastore.deactivate_user(user):
+                    count += 1
+            if count > 0:
+                flash(_('User(s) were successfully inactivated.'), 'success')
+        except Exception as exc:
```

do you really need to catch all exception?